src/MockeryTools/Http/HttpRequestOptionsMatcher.php

```diff
@@ -71,6 +81,30 @@ public function match(&$actual): bool
     }
 
 
+    /**
+     * @param mixed[] $actual
+     * @param mixed[] $expectedOptions
+     */
+    private function containsAllExpectedOptions(array $actual, array $expectedOptions): bool
+    {
+        foreach ($expectedOptions as $optionName => $optionValue) {
+            if (!isset($actual[$optionName])) {
+                return false;
+            }
+
+            if (!is_array($optionValue) && $actual[$optionName] !== $optionValue) {
+                return false;
+            }
+
```

Comment:
Pokud je $optionValue pole, pak musí být i $actual[$optionName] pole. V takovém případě by původní kód spadl na TypeErroru. Navrhuju přidat check:

```suggestion

            if (is_array($optionValue) && !is_array($actual[$optionName])) {
                return false;
            }

```